docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -0,0 +1,70 @@
+# Using Daemon Mode (Technical Preview) <!-- omit in toc -->
```

Comment:
This documentation gets too far into the technical details. I would focus on what is important to our user persona.
- Daemon Mode significantly improves the performance of Zowe CLI commands
- To try it out, first install the `next` version of Zowe CLI, obtain the native client from our [repo*](https://github.com/zowe/zowe-cli/releases) unzip it and add to your `PATH`. Then, to start the daemon, on Windows run `start zowe --daemon` and on Linux run `zowe --daemon &`. Finally, use `zowex` as your primary command instead of `zowe` (e.g. `zowex zosmf check status`).

`*` - we should include the `zowex` binary in our `next` bundle on our [Zowe Downloads](https://www.zowe.org/download.html) site as well.